tests/conftest.py

```diff
@@ -65,16 +131,22 @@ async def get_access_token(*args, **kwargs):
 
 
 @pytest.fixture(autouse=True)
-def clean_db(client):
-    # before each test, delete all existing requests from the DB
-    fake_jwt = "1.2.3"
-    res = client.get("/request", headers={"Authorization": f"bearer {fake_jwt}"})
-    assert res.status_code == 200
-    for r in res.json():
-        res = client.delete(
-            "/request/" + r["request_id"],
-            headers={"Authorization": f"bearer {fake_jwt}"},
-        )
+def clean_db():
+    """
+    Before each test, delete all existing requests from the DB
+    """
+    # The code below doesn't work because of this issue
+    # https://github.com/encode/starlette/issues/440, so for now reset
+    # using alembic.
+    # pytest-asyncio = "^0.14.0"
+    # from requestor.models import Request as RequestModel
+    # @pytest.mark.asyncio
+    # async def clean_db():
+    #     await RequestModel.delete.gino.all()
+    #     yield
+
```

Comment:
Please correct me if I'm wrong, but after the migration is done once, everytime `clean_db()` is called, will the `base` be the version with the `policy_id` automatically?

Reply:
no, the base is the first version, an empty database. and the head is the latest revision